frontend/src/concepts/pipelines/content/import/utils.ts

```diff
@@ -10,3 +10,10 @@ export enum PipelineUploadOption {
   URL_IMPORT,
   FILE_UPLOAD,
 }
+
+// Utility function to extract Kind from Pipeline YAML
+export const extractKindFromPipelineYAML = (yamlFile: string): string | undefined => {
```

Comment:
why not parse the yaml and then check if it is kind workflow instead of having this custom parsing logic